### ISSUE TITLE
Composer post create project commands fail on Windows

### ISSUE THREAD

**Issue by damsfx** [wintercms/winter#1242]
### Winter CMS Build

dev-develop

### PHP Version

8.1

### Database engine

MySQL/MariaDB

### Issue description

Related 
 - commit: https://github.com/wintercms/winter/commit/2b2d66c8e804b2b49a8bb640651151f52203d202 
 - #1241 

The new composer post commands on Windows are done with all default values.

My tests are running the following on both Windows and Windows Subsystem Linux: 
```bash
wintercms/winter (dev-develop 5f53526)
winter/storm (dev-develop a37c1f9)
winter/wn-backend-module (dev-develop e11d279)
winter/wn-cms-module (dev-develop 3ba7bcf)
winter/wn-system-module (dev-develop 9ac3b57)
```

- Windows = ❌ the command `php artisan winter:install` **is executed without user interaction**.
- WSL = ✅ All went well

### Steps to replicate

On a Windows system, simply run `composer create-project wintercms/winter example.com "dev-develop"`

**Comment by ghost** [comment]
It works on my windows system as the old composer-installation and like the the documentaion. 
When composer create-project wintercms/winter . "dev-develop" is done ypu can run winter:install and all works as know (select, database, etc)

**Comment by bennothommo** [comment]
@damsfx @nialma according to this - https://github.com/composer/composer/issues/11709#issuecomment-1801650122 - it won't be doable for `php artisan winter:install` to work on Windows as part of the post-create-project scripts.

Instead of @nialma's suggestion of rolling back the `post-create-project-cmd` changes in #1243, I wonder if we could opt instead to change it so it can bootstrap Winter non-interactively. Ideally, we can follow similar steps to what our [devcontainer bootstrap](https://github.com/wintercms/winter/blob/develop/.devcontainer/local-features/bootstrap-winter/bootstrap.sh) script does, namely:

- Run `php artisan winter:env`
- Run `php artisan key:generate`
- Create an empty `database.sqlite` file in the `storage` folder
- Update `.env` and set `DB_CONNECTION` as `sqlite` and `DB_DATABASE` as `storage/database.sqlite`
- Run `php artisan migrate`
- Run `php artisan winter:mirror public/`

This would get the environment set up in a way that it could be run immediately.

Any thoughts on the above? (/cc @LukeTowers)

**Comment by ghost** [comment]
@bennothommo 

 just did this for myself because I work with Windows and there is currently an error. With the "step back" in Composer I'm back to the "old" installation - if there's an easier way, I don't mind - it works fine for me under Windows....

Your idea is good, but not everyone works with SQLite... they should 
1. a query of the DB can be installed
2. DB name can be queried etc...
The current installation mode is really good

@damsfx i work also and at the moment only with windows an php artisan winter:install does ask me anything as the "old" versions with the "step back" in the composer.js

**Comment by bennothommo** [comment]
@nialma You're right, not everyone uses SQLite, but the intention is to make it so that `composer create-project wintercms/winter` can be used to basically get started with a Winter project immediately and then, *if necessary*, the user can modify the configuration / `.env` file to suit their environment.

You can see that Laravel does [exactly the same thing](https://github.com/laravel/laravel/blob/11.x/composer.json#L44-L48) when someone runs `composer create-project laravel/laravel`.

**Comment by ghost** [comment]
@bennothommo that sounds plausible and good....

> You can see that Laravel does [exactly the same thing](https://github.com/laravel/laravel/blob/11.x/composer.json#L44-L48) when someone runs composer create-project laravel/laravel.

and that is the reason why i use for a laravel-project a downloaded version and make it manual and not with composer...... local i use laragon with marianDB/mySQL php 8.2.24

**Comment by damsfx** [comment]
@bennothommo @nialma For myself, I use both MariaDB and SQLite databases, depending on the needs of the project.

Aligning with Laravel's choices doesn't worry me, as long as it allows new users to have a project up and running in a single command line.
Then, for those who want other settings, there's nothing to stop them running a `php artisan winter:install --force` command, right?

In my opinion, the more WinterCMS is aligned with Laravel, the better! Povilas' experience in testing Laravel-based CMS fits in with this idea.

**Comment by LukeTowers** [comment]
@bennothommo I'm game with your proposed flow, could we perhaps have the final step be outputting to the console instructions for running winter:install --force to make it easier to guide people to use the CLI installer if they've just created a new project so that they can go through the flow to set the other values handled by that process (user etc)?

**Comment by damsfx** [comment]
@LukeTowers @bennothommo I also get an error when running the command on Windows OS  
`https://github.com/php artisan winter:mirror public --relative` : 
> The environment file is invalid!
Failed to parse dotenv file. Encountered an unexpected escape sequence at ["storage\database.sqlite"].
Script @php artisan winter:mirror public --relative handling the post-create-project-cmd event returned with error code 1

So I've try to update the `.env` file to `DB_DATABASE="storage/database.sqlite"`, and then run again `pa winter:mirror public --relative` but it also fail.

> In WinterMirror.php line 208:
  symlink(): No such file or directory

The first values passed are : 
```php
$src = "../.htaccess";
$dest = "P:\_Sites\_Labs\new-winter-site\public/.htaccess";
```

All these post-installation commands are quite handy in a Linux environment, but under Windows they've become one of the worst experiences!

**Comment by LukeTowers** [comment]
@damsfx is it just `winter:mirror` with the `--relative` flag that causes it to fail?

**Comment by damsfx** [comment]
> [@damsfx](https://github.com/damsfx) is it just `winter:mirror` with the `--relative` flag that causes it to fail?

Sorry for this late request ... Maybe this issue can be closed due to the latest PRs on WinterCMS : https://github.com/wintercms/winter/pull/1265 and https://github.com/wintercms/winter/pull/1266

**Comment by LukeTowers** [comment]
Fixed by #1265 & #1266
